helpers/setup.py

```diff
@@ -17,13 +18,27 @@ def run(cls, config):
         """
 
         if not os.path.isdir("{}/.git".format(config["kobodocker_path"])):
+            # Move unique id file to /tmp in order to clone without errors
+            # (e.g. not empty directory)
+            uniqid_mv_command = [
+                "mv", Config.UNIQUE_ID_FILE,
+                "/tmp"
```

Comment:
Someday we should consider Windows support :wink:

Reply:
Yep! I've changed the code to be "future proof"